acceptancetests/requirements.txt

```diff
@@ -36,3 +36,4 @@ ipaddress>=1.0.16
 python-novaclient>=6.0.2
 pyOpenSSL>=17.5.0
 boto3>=1.5.18
+pyaml>=17.12.1
```

Comment:
why the dep change for a bundle change?

Reply:
When I set up locally to run the tests, the Python scripts threw an error due to missing this.

I used a clean virtualenv, which the tests don't use. This was probably missed due to (to-date) happening to have this in the global Python packages.